src/test/java/ay2122s1_cs2103t_w16_2/btbb/model/order/UniqueOrderListTest.java

```diff
@@ -35,8 +53,26 @@ public void contains_orderInList_returnsTrue() {
     }
 
     @Test
-    public void add_nullOrder_throwsNullPointerException() {
-        assertThrows(NullPointerException.class, () -> uniqueOrderList.add(null));
+    public void getTopTenClients_listContainsOrdersForElevenClients_returnsTopTenClients() {
+        List<Order> orderList = new ArrayList<>(List.of(ORDER_FOR_ALICE, ORDER_FOR_AMY, ORDER_FOR_BOB, ORDER_FOR_BENSON,
+                ORDER_FOR_CARL, ORDER_FOR_DANIEL, ORDER_FOR_ELLE, ORDER_FOR_FIONA, ORDER_FOR_GEORGE, ORDER_FOR_HOON,
+                ORDER_FOR_IDA));
+        LongStream.range(1, orderList.size() + 1).forEach(i -> {
+            LongStream.range(0, i).forEach(j -> uniqueOrderList.add(new OrderBuilder(orderList.get((int) i - 1))
+                    .withPrice(new Price(Integer.toString((int) i)))
+                    .build()));
+        });
```

Comment:
Maybe you can change your streams to loops since you are only using the numbers as counters/ indexes. A loops seems to communicate that more clearly than an arbitrary stream of numbers

```suggestion
        for (int i = 0; i < orderList.size(); i++) {
            for (int j = 0; j < i + 1; j++) {
                uniqueOrderList.add(
                        new OrderBuilder(orderList.get(i)).withPrice(new Price(Integer.toString(i + 1))).build());
            }
        }
```